### PR TITLE
adding funding blog post and logos

### PR DESCRIPTION
**Pull request by crazy4pi314**
Adding draft of post, scheduled for release on the 10th.

### CODE REVIEW

**Comment by crazy4pi314** [comment]
We may still have one more quote coming, so will be waiting to add it.

File: posts/_markdown/2021-member-announcement.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+title: Unitary Fund Announces New Support through New Membership Program
+date: January 10, 2022
+author: UF Team
+---
+
+Since 2018, Unitary Fund has supported the open quantum technology ecosystem around the world. We’ve awarded more than 50 [microgrants](https://unitary.fund/grants.html) to projects from more than 20 countries, developed the leading error-mitigating quantum [compiler](https://unitary.fund/mitiq.html), run [hackathons](https://unitaryfund.github.io/unitaryhack/), and supported open source community building.
+
+Today, Unitary Fund is proud to announce our new corporate member program. We thank these organizations for their support and look forward to their further contributions in the coming year.
+
+The corporate member program contributes to Unitary Fund’s work on a recurring, annual basis, and consists of two tiers: cCore members and supporting members. core members' contributions start at $100k annually and supporting member’s contributions start at $25k annually. Additionally, core members are a part of the Unitary Fund’s Strategic Board which reviews strategy, budget, impact, and recommendation’s quarterly.
```
Comment (nathanshammah) [inline posts/_markdown/2021-member-announcement.md:11]:
```suggestion
The corporate member program contributes to Unitary Fund’s work on a recurring, annual basis, and consists of two tiers: cCore members and supporting members. Core members' contributions start at $100,000 annually and supporting members' contributions start at $25,000 annually. Additionally, core members are a part of the Unitary Fund’s Strategic Board which reviews strategy, budget, impact, and recommendation’s quarterly.
```

File: posts/_markdown/2021-member-announcement.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+The Unitary Fund awards $4,000 grants to individuals and teams pursuing projects that leverage quantum technologies to benefit humanity. Projects could be open source quantum software, educational materials and workshops, a new quantum sensor prototype, or much more. The program is unique in its inclusivity (no credentials are required to apply) and low friction (awards are no-strings-attached).
+
+## Unitary Fund Supporting Members: Xanadu, IonQ, BCG, Pasqal, IQM, and DoraHacks
+
+![](../images/supporting-members-2021.png "Unitary Fund Supporting Members: Xanadu, IonQ, BCG, Pasqal, IQM, and DoraHacks")
+
+"It was important for Pasqal to reaffirm its support to open-source initiatives in quantum software by becoming a sponsor of Unitary Fund” says Loic Henreit, CTO at France-based quantum computing startup Pasqal. “We are thrilled to join the efforts led by Unitary Fund to build a solid ecosystem in this field."
```
Comment (nathanshammah) [inline posts/_markdown/2021-member-announcement.md:25]:
```suggestion
"It was important for Pasqal to reaffirm its support to open-source initiatives in quantum software by becoming a sponsor of Unitary Fund” says Loïc Henriet, Chief Technology Officer of Pasqal, a French startup building quantum processors out of neutral atoms. “We are thrilled to join the efforts led by Unitary Fund to build a solid ecosystem in this field."
```

File: posts/_markdown/2021-member-announcement.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+"We are delighted to support the Unitary Fund's mission. They are doing first-class work building out a quantum ecosystem for everyone. You can see the Unitary Fund's impact everywhere you look," said Nathan Killoran, Head of Software, Algorithms, & Quantum Machine Learning at Xanadu, a photonic quantum computer company in Canada.
+
+Quantum computers have the potential to be revolutionary for many industries with its capability to develop efficient solutions for computational problems beyond the scope of a conventional machine and uncover unimaginable new possibilities.
```
Comment (nathanshammah) [inline posts/_markdown/2021-member-announcement.md:29]:
```suggestion
```

Comment (nathanshammah) [reply posts/_markdown/2021-member-announcement.md:29]: If the suggestion to cut is not accepted, "its" should eventually be changed to "their"

**Review by nathanshammah — CHANGES_REQUESTED** [review]

**Comment by nathanshammah** [comment]
I made suggestions to the markdown file only. I also suggest a change in the blog file name, from `2021-member-announcement` to `2021-corporate-members`.

File: posts/_markdown/2021-member-announcement.md (hunk 3 above)
Comment (willzeng) [reply posts/_markdown/2021-member-announcement.md:29]: I'd support this cut

**Review by willzeng — CHANGES_REQUESTED** [review]
Let's remove the reference to "Corporate Members" and just say Members since non-corporates would be more than fine as well.

File: posts/2021-corporate-members.html [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+                                <p>Since 2018, Unitary Fund has supported the open quantum technology ecosystem around the world. We’ve awarded more than 50 <a href="https://unitary.fund/grants.html">microgrants</a> to projects from more than 20 countries, developed the leading error-mitigating quantum <a href="https://unitary.fund/mitiq.html">compiler</a>, run <a href="https://unitaryfund.github.io/unitaryhack/">hackathons</a>, and supported open source community building.</p>
+                                <p>Today, Unitary Fund is proud to announce our new corporate member program. We thank these organizations for their support and look forward to their further contributions in the coming year.</p>
+                                <p>The corporate member program contributes to Unitary Fund’s work on a recurring, annual basis, and consists of two tiers: cCore members and supporting members. Core members’ contributions start at $100,000 annually and supporting members’ contributions start at $25,000 annually. Additionally, core members are a part of the Unitary Fund’s Strategic Board which reviews strategy, budget, impact, and recommendation’s quarterly.</p>
```
Comment (willzeng) [inline posts/2021-corporate-members.html:82]:
```suggestion
                                <p>The corporate member program contributes to Unitary Fund’s work on a recurring, annual basis, and consists of two tiers: Core members and Supporting members. Core members are a part of the Unitary Fund’s Strategic Board which reviews strategy, budget, impact, and recommendation’s quarterly.</p>
```

**Review by willzeng — DISMISSED** [review]
@crazy4pi314 I made some final edits to the blog. This is now ready to go live IMO.

It can either go live tomorrow (Friday) which is the date it has now, or if you think it better to wait until Monday then let's do that.

**Comment by crazy4pi314** [comment]
This is ready now @nathanshammah, I'll merge on monday if you haven't sooner.

**Pull request merged**